beanprice/sources/yahoo.py

```diff
@@ -70,10 +71,40 @@ def parse_currency(result: Dict[str, Any]) -> str:
 }
 
 
+def get_price_series(ticker: str,
+                     time_begin: datetime,
+                     time_end: datetime) -> List[Tuple[datetime, Decimal]]:
```

Comment:
```suggestion
                     time_end: datetime) -> Tuple[List[Tuple[datetime, Decimal]], str]:
```

Reply:
Good catch. Done. Will upload and merge.
I'll eventually integrate pytype() in the Bazel build, so it'll run automatically.